curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/65672136535209761a5cf02b.md

```diff
@@ -13,10 +13,22 @@ Don't forget you need to interpolate with the dollar sign here.
 
 # --hints--
 
-You should add an `onclick` with the value `playSong(${song.id})` as the second attribute and value of the `button` element.
+You do not need to change the other parts of the code. All you need to do is add the `onclick` attribute to the button element.
```

Comment:
I'm not crazy about this sentence. I think if we fix the tests so the order doesn't matter we can just simplify the original sentence.

```suggestion
You should add an `onclick` attribute with the value `playSong(${song.id})` to the `button` element.
```

Reply:
I think all we care about here is that the handler was added correctly. Then we can test that the button element still has the class on it in the next test.